test_app/views.py

```diff
@@ -1,30 +1,35 @@
 from rest_framework import permissions
 from rest_framework.viewsets import ModelViewSet
 
+from ansible_base.lib.utils.views.ansible_base import AnsibleBaseView
 from test_app import serializers
-from test_app.models import RelatedFieldsTestModel
+from test_app.models import EncryptionModel, Organization, RelatedFieldsTestModel, Team, User
 
 
-class TestAppViewSet(ModelViewSet):
+class TestAppViewSet(ModelViewSet, AnsibleBaseView):
     permission_classes = [permissions.IsAuthenticated]
 
     def get_queryset(self):
         return self.serializer_class.Meta.model.objects.all()
 
 
 class OrganizationViewSet(TestAppViewSet):
+    queryset = Organization.objects.all()
```

Comment:
this might be a rebase conflict, but you don't need a `queryset` because the parent viewset defines `get_queryset`.

Reply:
Removed.

Reply:
Actually, upon reviewing the code more thoroughly, the ViewSets will require queryset because the router itself is utilizing that field. And we can't call get_queryset because we have a View class, not an instance.

Reply:
Ultimately, as per RBAC core requirements, we can't produce a queryset without a view and a user.